internal/component/postgresql/postgresdbaccess.go

```diff
@@ -270,15 +260,15 @@ func (p *PostgresDBAccess) Get(parentCtx context.Context, req *state.GetRequest)
 	var (
 		value    []byte
 		isBinary bool
-		etag     uint32
+		etag     sql.NullInt64
```

Comment:
I think pgx has a specific nullable type: https://pkg.go.dev/gopkg.in/jackc/pgx.v2#hdr-Null_Mapping